website/docs/docs/collaborate/govern/model-contracts.md

```diff
@@ -76,9 +76,18 @@ When building a model with a defined contract, dbt will do two things differentl
 
 ## FAQs
 
+### Where are contracts supported?
+
+At present, model contracts are supported for:
+- SQL models (not yet Python)
+- Models materialized as `table`, `view`, and `incremental` (with `on_schema_change: append_new_columns`)
+- On the most popular data platforms — but which `constraints` are supported/enforced varies by platform
```

Comment:
i'd recommend being more clear on which platforms are supported here. what are 'popular ones'? also should we link to the supported data platform page?

```suggestion
- Most commonly used [data platforms](/docs/supported-data-platforms), such as Snowflake, BigQuery, Databricks, Redshift, and so on, support `constraints`. However, the specific `constraints` that are supported or enforced can vary depending on the platform.
```